wiki/en/Running-a-Server.md

```diff
@@ -150,21 +141,17 @@ Note also that mean ADSL2 transfer rate is 10 Mbit/s for downstream and 1 Mbit/s
 
 <figure><img src="{% include img/en-screenshots/server-window-setup.inc %}" style="width:80%; border:5px solid grey;" loading="lazy" alt="Image of the Jamulus server setup window"></figure>
 
-### Make My Server Public
+### The Directory list 
 
-By default, you will be running a Private Server and need to [read these instructions](#running-a-private-server) to have others connect to you in this mode.
+**None**: By default, you will not be connected to a directory and will be running in unregistered mode. [Read these instructions](#running-an-unregistered-server) to have other people connect to your server in this mode.
```

Comment:
```suggestion
**None**: By default, you will not be connected to a directory and will be running in unregistered mode. This means your server will not be publically visible. [Read these instructions](#running-an-unregistered-server) to have other people connect to your server in this mode.
```

Reply:
I think attempts to resurrect the ideas of public and private may not be right. But let's see. You can run an unregistered server and make it public by putting its address out in some other way, for instance.

Reply:
You can, but then you know that you're publishing it. I think this discussion should go to https://github.com/jamulussoftware/jamulus/discussions/2228

Reply:
>  This means your server will not be publically visible.

I wouldn't bother with the addition.  Simply registering doesn't mean others will see you.  You'd get into the discussion of different types of Directory here, which isn't wanted.